public/stylesheets/style.css

```diff
@@ -58,14 +59,66 @@ a {
   background-color: #fff;
   vertical-align: -7px;
 }
-.badge-roster {
-
+*/
+.badge-small- {
+  display: inline-block;
+  min-width: 10px;
+  padding: 2px 4px;
+  font-size: 12px;
+  font-weight: 700;
+  line-height: 1;  
+  text-align: center;
+  white-space: nowrap;
+  vertical-align: baseline;
+  border-radius: 50%;
+  display: inline-block;
+  width: 25px;
+  height: 25px;
+  margin: 0 2px 0 5px;    
+  content: url(https://cdn4.iconfinder.com/data/icons/ios7-essence/22/editor_clipboard_checkmark_done-512.png) no-repeat 0 0;
+  background-size: 100%;
+  background-color: #fff;
+  vertical-align: -7px;
 }
-.badge-leaders {
-
+.badge-small-leaders {
```

Comment:
these may change to match image so im going to leave for now. i.e. the same styles might not work for the different images